.travis.yml

```diff
@@ -1,18 +1,25 @@
 language: ruby
+dist: trusty
 
 rvm:
-  - 2.2
+  # - "2.2"
+  - "2.3.1"
 
-branches:
-  only:
-  - master
-  - develop
-  - deploy-release
-  - /^issue.*$/
+gemfile:
+  - cf.Gemfile
+
+# branches:
```

Comment:
It was commented out for debugging, but I think it should probably be removed to allow rake tests to run on feature branches during development.  Any thoughts on this, @rolivieri ?